src/Header/Header.js

```diff
@@ -15,9 +18,29 @@ class Header extends Component {
                     <FontAwesomeIcon icon="dove" className="dove" /> GiveAway 
                 </div>
                 <SearchBox />
+                <div className="user">
+                    <FontAwesomeIcon icon="user-circle" className="icon_user" /> 
+                    <p className="sign_in" onClick={()=> this.props.toggleModal(true) }>Sign in</p>
+                </div>
             </div>
         )
     }
 }
 
-export default Header;
\ No newline at end of file
+function mapStateToProps(state) {
+    return{
```

Comment:
?

Reply:
I thought this `mapStateToProps` must be there in order to use `mapDispatchToProps`